### PR TITLE
CI: Automatic vX tag update workflow

### PR DESCRIPTION
**Pull request by tooomm**
See https://github.com/lycheeverse/lychee-action/issues/169#issuecomment-1454895532

Let's try how it works!

### CODE REVIEW

File: .github/workflows/versioning.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: Keep major version tags up-to-date
+
+on:
+  release:
+    types: [published, edited]
+
+jobs:
+  actions-tagger:
+    runs-on: windows-latest
```
Comment (mre) [inline .github/workflows/versioning.yml:9]: The Windows runners are significantly slower than the Linux runners in my experience. Can we change that to `ubuntu-latest`? 
It [should work](https://github.com/bcanseco/github-contribution-graph-action/blob/31a07d74b3430073cd5dd4fcc7238464e9f7d0d9/.github/workflows/release.yml).

Comment (tooomm) [reply .github/workflows/versioning.yml]: You are right, it does: https://github.com/bcanseco/github-contribution-graph-action/actions/runs/3981711788/workflow

Thanks for linking the example. I was already wondering why windows was particularly used in the example...

**Comment by mre** [comment]
Thanks!

**Comment by mre** [comment]
Let's try it with the next release. ⭐

**Comment by tooomm** [comment]
>Let's try it with the next release. ⭐

If you edit the last release, it should also trigger the action!

**Comment by mre** [comment]
Hum, I went to https://github.com/lycheeverse/lychee-action/releases/edit/v1.6.1, changed the release description and hit `Update Release`. The pipeline didn't run for some reason. 🤔 Maybe I misunderstand what "editing a release" means and changing the text is not enough.

**Comment by tooomm** [comment]
That is exactly what the docs says as well:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release

Webhook events and payloads link:
image

<br>

They even link to an example telling you how to adjust the description in the web UI. :)

As long as you were not dealing with a draft, it should have triggered I think. 🤷

**Comment by mre** [comment]
Late feedback, but it worked with the latest 1.7.0 release. 🚀 
Logs here: https://github.com/lycheeverse/lychee-action/actions/runs/4726601856/jobs/8386343900
Thanks @tooomm.

**Pull request merged**